lang/de.yml

```
@@ -108,14 +108,14 @@ LOOKUP_BLOCK: "{0} {platziert|entfernt} {1}."
 LOOKUP_CONTAINER: "{0} {hinzugefügt|entfernt} {1} {2}."
 LOOKUP_HEADER: "{0} Ergebnisse"
 LOOKUP_INTERACTION: "{0} {angeklickt|getötet} {1}."
-LOOKUP_ITEM: "{0} {abgeholt|abgeworfen} {1} {2}."
-LOOKUP_LOGIN: "{0} eingeloggt {rein|raus}."
+LOOKUP_ITEM: "{0} {aufgehoben|abgeworfen} {1} {2}."
```

I think "weggeworfen" is a more natural translation for "dropped" in this context

```suggestion
LOOKUP_ITEM: "{0} {aufgehoben|weggeworfen} {1} {2}."
```